{{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/assets/index.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
+{% block head %}
```

Comment:
I think we can drop all of this `{% block head %}` section once the table is bootstrappin